RawWaterManagement/ObservedAsset/examples/example-normalized.jsonld

```diff
@@ -0,0 +1,93 @@
+{
+  "location": {
+    "type": "Property",
+    "value": {
+      "type": "Point",
+      "coordinates": [
+        62.8746225,
+        169.606193
+      ]
+    }
+  },
+  "address": {
+    "type": "Property",
+    "value": {
+      "streetAddress": "Attorney with impact teacher program near. System thus tonight interest three several.",
+      "addressLocality": "Use source green the. Hit beat its return cost mother.",
+      "addressRegion": "Section between apply along despite consider itself they. Need husband term. Check situation soon firm.",
+      "addressCountry": "Conference artist senior exactly close. Lay popular sort interest little together. Information ability information.",
+      "postalCode": "Research population group situation cell.",
+      "postOfficeBoxNumber": "Decision organization leg. Brother particularly physical war material task. Safe action skill condition reflect anyone special.",
+      "areaServed": "World institution raise budget. Late think over."
+    }
+  },
+  "areaServed": {
+    "type": "Property",
+    "value": "Response wish training arrive dark."
+  },
+  "id": "urn:ngsi-ld:dataModel:id:URJR:40654097",
+  "dateCreated": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "2004-11-25T21:28:39Z"
+    }
+  },
+  "dateModified": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "2012-12-07T06:42:31Z"
+    }
+  },
+  "source": {
+    "type": "Property",
+    "value": "Treat put herself. Try task single Republican visit."
+  },
+  "name": {
+    "type": "Property",
+    "value": "Kid do lead investment step even time. Parent agreement behind despite they operation whether majority. Daughter nice exist cut despite fall."
+  },
+  "alternateName": {
+    "type": "Property",
+    "value": "Action sea side. Few huge course already."
+  },
+  "description": {
+    "type": "Property",
+    "value": "Prevent hold the. Firm join kid. Generation firm run whom future situation."
+  },
+  "dataProvider": {
+    "type": "Property",
+    "value": "Story future former degree full program north. Determine trouble kid budget."
+  },
+  "owner": {
```

Comment:
type suggested to be Relationship
note:
1. Relationships should not be modelled as properties, otherwise you completely lost the interest of NGSI-LD which is based on graph. 
2. Arrays in NGSI-LD are stored as strings so you can not use the API to query indiviudual alements in a array. 
3. Relationships can not be provided within arrays.

When you need to have several relationships with the same name, then you have to use the datasetId 

Example for a relationship where one entity would "monitor" 2 different rooms. It thus have 2 relationshipsnames "monitored" directed toward room01 and room02. The datasetId allows to differentiate the 2 relationships.
"monitors":[
	{
	   "type":"Relationship",
	   "object":"urn:ngsi-ld:Room:01",
	   "datasetId":"urn:ngsi-ld:dataSet:01"},
	},
	"type":"Relationship",
	"object":"urn:ngsi-ld:Room:02" ,
	"datasetId":"urn:ngsi-ld:dataSet:02"}
	]